tests/unit/test_app.py

```diff
@@ -1316,7 +1316,7 @@ def test_http_request_to_dict_is_json_serializable(http_request_kwargs):
     request_dict = request.to_dict()
     # We should always be able to dump the request dict
     # to JSON.
-    assert json.dumps(request_dict, default=handle_decimals)
+    assert json.dumps(request_dict, cls=DefaultJSONEncoder)
```

Comment:
Could we get another test added that injects a custom encoder and does something with it that the default encoder cannot do?